template/pipelines/train.py

```diff
@@ -0,0 +1,62 @@
+# {% include 'template/license_header' %}
+
+
+from steps import evaluate_model, finetune, prepare_data, promote
+from zenml import logging as zenml_logging
+from zenml import pipeline
+
+zenml_logging.STEP_LOGS_STORAGE_MAX_MESSAGES = (
+    10000  # workaround for https://github.com/zenml-io/zenml/issues/2252
+)
```

Comment:
shouldn't we just disable step logs instead for the specific steps?

Reply:
It is already off, so I can just remove this one.